dvc/progress.py

```diff
@@ -8,28 +8,7 @@
 from dvc.utils import env2bool
 
 logger = logging.getLogger(__name__)
-
-
-class TqdmThreadPoolExecutor(ThreadPoolExecutor):
-    """
-    Ensure worker progressbars are cleared away properly.
-    """
-
-    def __enter__(self):
-        """
-        Creates a blank initial dummy progress bar if needed so that workers
-        are forced to create "nested" bars.
-        """
-        blank_bar = Tqdm(bar_format="Multi-Threaded:", leave=False)
-        if blank_bar.pos > 0:
-            # already nested - don't need a placeholder bar
-            blank_bar.close()
-        self.bar = blank_bar
-        return super(TqdmThreadPoolExecutor, self).__enter__()
-
-    def __exit__(self, *a, **k):
-        super(TqdmThreadPoolExecutor, self).__exit__(*a, **k)
-        self.bar.close()
+TqdmThreadPoolExecutor = ThreadPoolExecutor
```

Comment:
Do we need this anymore?

Reply:
For now I think it's best to leave this direct definition just in case we want to go back to having a wrapper. The current choices of verbosity means we don't need a `Tqdm` wrapper for `ThreadPoolExecutor` but I feel this may change in future...

Reply:
What does verbosity have with this?

Reply:
I agree with @Suor , doesn't look like there is any point in keeping TqdmThreadPoolExecutor as an alias. If we ever need to bring it back, we'll just rename it where needed in those few places that it is used.